### ISSUE TITLE
x86 problem (i think)

### ISSUE THREAD

**Issue by RickardPettersson** [charlesw/tesseract-ocr-dotnet#6]
Hello, i whant to start with thank you for do tesseract-ocr for .Net 4.0!

Everything has worked good on my developer machine but now when i should publish the project that are a ASP.NET MVC 3 project on my webserver its seems that something is wrong.

I getting this error: http://pastebin.com/LUvBT6ak

I have the libtesseract.dll in the bin folder like it should on the server and everything looks like it should what i think, but when i searching for this problem on google i find this issue on the old project for tesseract-ocr on .Net: http://code.google.com/p/tesseractdotnet/issues/detail?id=1

And it looks like they think it is x86 vs x64 problem, so i should whant you to release the dll as a x86 and a x64 binaries or if you have any other idea?

I can not open the code becouse of i dont have c++ visual studio and stuff else i should try todo it my self.

**Comment by charlesw** [comment]
Yep you hit the nail on the head there mate, x64 isn't supported atm, it should be possible to add it as Tesseract-ocr does support x64 at least on linux anyway but it could be a while till I get time to add it. Your best bet if you still want to use this library is expose your the OCR functionality as a WCF service (or similar) run as a Windows Service.

**Comment by RickardPettersson** [comment]
Sorry i just find that c++ runtime not was installed on my webserver.

**Comment by charlesw** [comment]
OK yes that would cause similar issues but you still shouldn't be able to run when the IIS App Pool is running in x64 mode.
